src/tests/Interop/Swift/SwiftAbiStress/SwiftAbiStress.cs

```diff
@@ -0,0 +1,6683 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.Swift;
+using Xunit;
+
+public class SwiftAbiStress
```

Comment:
Let's add this test to the Interop.csproj project directly and use the following attributes to do the correct test filtering (instead of issues.targets and the csproj filtering).

You'll also need to add a CMakeProjectReference to the CMakeLists..txt in this folder from Interop.csproj.

```suggestion
[PlatformSpecific(TestPlatforms.OSX | TestPlatforms.iOS | TestPlatforms.tvOS)]
[ActiveIssue("https://github.com/dotnet/runtime/issues/94081", TestRuntimes.Mono)]
public class SwiftAbiStress
```

Reply:
Does it look good now?
I assume we'll want to put the other Swift tests under the same plan too (although let's do it separately from this PR).

Reply:
The change looks great, but it looks like there's some issues with a folder that only has targets on some platforms. I don't want to make you do the infra work to make this work correctly, so you can revert these changes.

We'll do it (for these tests and the other Swift tests) in a follow-up PR.